tests/unit_tests/test_wave_06.py

```diff
@@ -50,33 +46,33 @@ def test_best_by_category_with_duplicates():
     assert best_item.category == "Clothing"
     assert best_item.condition == pytest.approx(4.0)
 
-@pytest.mark.skip
+
+# @pytest.mark.skip
 def test_swap_best_by_category():
     # Arrange
     # me
     item_a = Decor(condition=2.0)
     item_b = Electronics(condition=4.0)
     item_c = Decor(condition=4.0)
-    tai = Vendor(
-        inventory=[item_a, item_b, item_c]
-    )
+    tai = Vendor(inventory=[item_a, item_b, item_c])
 
     # them
     item_d = Clothing(condition=2.0)
     item_e = Decor(condition=4.0)
     item_f = Clothing(condition=4.0)
-    jesse = Vendor(
-        inventory=[item_d, item_e, item_f]
-    )
+    jesse = Vendor(inventory=[item_d, item_e, item_f])
 
     # Act
     result = tai.swap_best_by_category(
-        other=jesse,
-        my_priority="Clothing",
-        their_priority="Decor"
+        other=jesse, my_priority="Clothing", their_priority="Decor"
     )
+    # Assert
+    assert result
+    assert (len(tai.inventory) and len(jesse.inventory)) == 3
+    assert (item_f and item_a and item_b) in tai.inventory
+    assert (item_c and item_d and item_e) in jesse.inventory
```

Comment:
These checks are not working how you might expect. First `(item_f and item_a and item_b)` is evaluated, which will return a single Item object, the last one in the list. Then just that one Item object will be considered in the remainder of the expression `<Item> in <name>.inventory`. 

If we want to condense the checks for all items to a single line, one approach would be to convert the lists to sets and use set operations:
```python
assert {item_f, item_a, item_b}.issubset(tai.inventory)
assert {item_c, item_d, item_e}.issubset(jesse.inventory)
```
We could also use `all`:
```python
assert all(item in tai.inventory for item in [item_f, item_a, item_b])
assert all(item in jesse.inventory for item in [item_c, item_d, item_e])
```